core/quantity/src/test/Formatting.test.ts

```diff
@@ -276,7 +287,7 @@ describe("Formatting tests:", () => {
     const testFormat = new Format("test");
 
     await testFormat.fromJSON(unitsProvider, json).catch(() => {
-      assert.isTrue(false);
+      expect.fail();
     });
```

Comment:
```suggestion
    await expect(testFormat.fromJSON(unitsProvider, json)).resolves.not.toThrow();
```

Reply:
if you think this looks good, you can replace other similar instances in this file

Reply:
I do prefer that. Updated thanks 😄